ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -122,9 +134,21 @@ internal class CustomerCenterViewModelImpl(
         val customerInfo = purchases.awaitCustomerInfo(fetchPolicy = CacheFetchPolicy.FETCH_CURRENT)
 
         // Customer Center WIP: update when we have subscription information in CustomerInfo
-        val activeEntitlement = customerInfo.entitlements.active.isEmpty()
+        val activeEntitlement = customerInfo.entitlements.active.isNotEmpty()
         if (activeEntitlement) {
-            return CustomerCenterConfigTestData.purchaseInformationMonthlyRenewing
+            val entitlement = customerInfo.entitlements.active.values.first()
```

Comment:
We're reading `entitlements.active` twice here, but we can't be sure that it's still not empty by the time we call `first()`. (Maybe we can be sure with the current implementation of `CustomerInfo`, but we never know how it will change in the future.)

We can also do something like this: 

```kotlin
val entitlement = customerInfo.entitlements.active.values.firstOrNull()
if (entitlement != null) {
    // ...
}
```

Reply:
Sure. This is temporary btw, the logic requires we add `subscriptionsByProductIdentifier` to `CustomerInfo` so I am just doing the dumb logic here. But will change that to prevent weird issues

Reply:
Ah nice okay, thanks for the context!